opm/input/eclipse/Schedule/Group/Group.cpp

```diff
@@ -680,27 +680,31 @@ void Group::delGroup(const std::string& group_name) {
         };
 }
 
-bool Group::update_gefac(double gf, bool transfer_gf) {
+bool Group::update_gefac(double gf, bool use_efficiency_in_network_arg) {
     bool update = false;
     if (this->gefac != gf) {
         this->gefac = gf;
         update = true;
     }
 
-    if (this->transfer_gefac != transfer_gf) {
-        this->transfer_gefac = transfer_gf;
+    if (this->use_efficiency_in_ext_network != use_efficiency_in_network_arg) {
+        this->use_efficiency_in_ext_network = use_efficiency_in_network_arg;
         update = true;
     }
 
     return update;
 }
 
-double Group::getGroupEfficiencyFactor() const {
+double Group::getGroupEfficiencyFactor(bool network) const {
```

Comment:
`network` probably should be renamed as `extended_network`, since it only matters for extended_network.

Reply:
Aim to support this also in standard networks now.